Sources/NIOCore/AsyncSequences/NIOAsyncSequenceProducer.swift

```diff
@@ -59,12 +64,12 @@ public protocol NIOAsyncSequenceProducerDelegate: Sendable {
     /// Termination happens if:
     /// - The ``NIOAsyncSequenceProducer/AsyncIterator`` is deinited.
     /// - The ``NIOAsyncSequenceProducer`` deinited and no iterator is alive.
-    /// - The subscriber's `Task` is cancelled.
+    /// - The consuming `Task` is cancelled (e.g. `for await let element in`).
     /// - The source finished and all remaining buffered elements have been consumed.
     func didTerminate()
 }
 
-/// This is an ``AsyncSequence`` that supports a unicast ``AsyncIterator``.
+/// This is an ``Swift/AsyncSequence`` that supports a unicast ``Swift/AsyncIterator``.
```

Comment:
FYI: These will cause you docc warnings now; your call if you want to have them warn or just not do such links for now.

Reply:
I know, we have these already in various places of NIO docs.